src/rewriter/README.md

```diff
@@ -94,17 +112,26 @@ processing are controlled by `rewriter.py` and `parser.py`.
 The symbols usable in `(<id> <expr>+)` can be found in `node.py`. The sorts
 available can be found in `mkrewrites.py`.
 
-
 ## Changes from Previous Works
 
 In comparison to
 [fmcad22](https://ieeexplore.ieee.org/abstract/document/10026573), we have
-removed the `let` and `cond` expressions in favour of more atomic rewrite rules.
-`let` is replaced by `def` which allow symbols to be shared across the
-condition, source, and target terms.
-
-We also deleted the `const` modifier since the evaluation of constant
-expressions should be handled elsewhere.
+removed the `let` expression.  `let` is replaced by `def` which allow symbols to
+be shared across the condition, source, and target terms.
+
+We also deleted the `const` modifier the rationale is that there are 3 reasons to use `:const`:
+1. There should not be a rule that is only sound when its arguments are constant, so `:const` should not exist in this case.
```

Comment:
I don't quite understand this? Add why there shouldn't (I guess because the evaluator would do this)? What changed so that we don't need this anymore (evaluater again)? Isn't this a subcase of 2?

Reply:
The logical validity of a rule shouldn't be impacted by whether something's const or not

Reply:
Ah okay that makes sense, thanks.